consensus/src/consensusdb/mod.rs

```diff
@@ -187,4 +191,34 @@ impl ConsensusDB {
         iter.seek_to_first();
         Ok(iter.collect::<Result<HashMap<HashValue, QuorumCert>>>()?)
     }
+
+    pub fn save_node(&self, node: &Node) -> Result<(), DbError> {
+        let batch = SchemaBatch::new();
+        batch.put::<NodeSchema>(&node.digest(), node)?;
+        self.commit(batch)?;
+        Ok(())
+    }
+
+    pub fn save_certified_node(&self, node: &CertifiedNode) -> Result<(), DbError> {
+        let batch = SchemaBatch::new();
+        batch.put::<CertifiedNodeSchema>(&node.digest(), node)?;
+        self.commit(batch)?;
+        Ok(())
+    }
+
+    pub fn get_certified_nodes(&self) -> Result<HashMap<HashValue, CertifiedNode>, DbError> {
+        let mut iter = self
+            .db
+            .iter::<CertifiedNodeSchema>(ReadOptions::default())?;
+        iter.seek_to_first();
+        Ok(iter.collect::<Result<HashMap<HashValue, CertifiedNode>>>()?)
+    }
+
+    pub fn delete_certified_nodes(&self, digests: Vec<HashValue>) -> Result<(), DbError> {
```

Comment:
do we need a range delete function to delete nodes after a state sync catch up? and also for epoch changes?

Reply:
since it's not keyed by rounds but digests, there's not really a `range_delete`